src/sentry/integrations/mixins/issues.py

```diff
@@ -99,9 +121,15 @@ def get_group_link(self, group, **kwargs):
 
     def get_group_description(self, group, event, **kwargs):
         output = self.get_group_link(group, **kwargs)
-        body = self.get_group_body(group, event)
-        if body:
-            output.extend(["", "```", body, "```"])
+        if isinstance(event, GroupEvent) and event.occurrence is not None:
```

Comment:
I think event could be None in `get_create_issue_config`

Reply:
added an extra check that it exists